mwaa/bucket/dags/test/test_emr.py

```
@@ -0,0 +1,46 @@
+import logging
+from datetime import datetime
+
+from airflow import DAG
+from modules.transformer import (
+    create_check_emr_termination_task,
+    create_execute_emr_task,
+    create_terminate_emr_cluster_task,
+)
+from utils.xcom import create_push_to_xcom_task, pull_from_xcom
```

Ruff (0.8.2)

`utils.xcom.pull_from_xcom` imported but unused

Remove unused import: `utils.xcom.pull_from_xcom`

(F401)